incubator/mattermost-team-edition/templates/ingress.yaml

```diff
@@ -0,0 +1,47 @@
+{{- $ingress := .Values.ingress -}}
+{{- if $ingress.enabled -}}
+{{- $serviceName := include "mattermost-team-edition.name" . -}}
+{{- $servicePort := .Values.service.externalPort -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "mattermost-team-edition.fullname" . }}
+  labels:
+    chart: {{ template "mattermost-team-edition.chart" . }}
+    app: {{ template "mattermost-team-edition.fullname" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
```

Comment:
Don't set nginx-ingress specific annotations by default. Instead provide an option to configure annotations. You may add these defaults to `values.yaml` and comment them out.

Reply:
done, thanks!